act-rules-format.bs

```diff
@@ -296,6 +296,19 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </div>
 
+<div class=example>
+  <p>**Example:** The applicability of a rule with the page as a test target</a></p>
+  <blockquote>
+    <p>The rule applies to any page where the document element is an `html` element not embedded in another page, for example as part of an `iframe` or `object`.</p>
+  </blockquote>
+</div>
+
+<div class=example>
+  <p>**Example:** The applicability of a rule with a DOM attribute as a test target</a></p>
+  <blockquote>
+    <p>Any `role` attribute that is specified on an HTML or SVG element.</p>
```

Comment:
I suggest changing 

"Any `role` attribute that is specified on an HTML or SVG element."

-->

"The rule applies to any any `role` attribute that is specified on an HTML or SVG element."

... to align better with the other example.

Reply:
@WilcoFiers, is "HTML element" and "SVG element" unambiguous enough for this example in light of our discussion yesterday on whether HTML element meant "element in the HTML namespace", "listed in the HTML specification" or "has its semantics defined in the HTML specification"?